pkg/util/liveconfig/dev.go

```diff
@@ -64,10 +65,23 @@ func (d *dev) DefaultInstallerPullSpecOverride(ctx context.Context) string {
 	return os.Getenv(hiveDefaultPullSpecEnvVar)
 }
 
-func (p *dev) AdoptByHive(ctx context.Context) (bool, error) {
+func (d *dev) AdoptByHive(ctx context.Context) (bool, error) {
 	adopt := os.Getenv(hiveAdoptEnableEnvVar)
 	if adopt != "" {
 		return true, nil
 	}
 	return false, nil
 }
+
+func (d *dev) OCMValidClientIDs() ([]string, error) {
```

Comment:
This looks like the same function as in prod. It might be worth defining this function separately from the methods and then calling the same function in both dev and prod.

Reply:
It is only the same function for now. Once we introduce proper live config Prod will be different from Dev in that Dev will continue to pull from the ENV and Prod will pull from a KeyVault. I'll extract it for now to reduce the duplicated code.